addons/web/static/src/core/popover/popover.js

```diff
@@ -89,7 +89,15 @@ Popover.props = {
         type: Function,
         optional: true,
     },
-    target: HTMLElement,
+    iframe: { type: HTMLIFrameElement, optional: true },
+    target: {
+        validate: (target) => {
+            // target may be inside an iframe, so get the Element constructor
+            // to test against from its owner document's default view
+            const Element = target?.ownerDocument?.defaultView.Element;
```

Comment:
Worth noting that in Chrome if an element in appended into an iframe, the constructor doesn't get reassigned and so this check will fail. In practice unless an owl instance outside the iframe mounts an app inside the iframe, this shouldn't ever happen but worth thinking about.

Reply:
Thanks ;-)
I was not aware of that!